single_level.py

```diff
@@ -90,22 +90,67 @@ def makeRow(resource):
 		else:
 			row.append("false")
 	
-	print row
+	print row	
 
 def main():
-	print "Creating a csv"
+	#User input to refine functionality of script 
+	print "Welcome to DACSspace!"
+	print ""
+	print "I'll ask you a series of questions to refine how the script works."
+	print "If you want to use the default value for a question press the ENTER key."
+	print ""
+	unpublished_response = raw_input("Do you want DACSspace to include unpublished resources? (default is no): ")
+	uniqueid_response = raw_input("Do you want to further limit the script by a specific resource id? (default is no): ")
+	
+	#Getting list of resources
+	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
+	
+	#Creating csv
 	writer = csv.writer(open(spreadsheet, "wb"))
-	column_headings = ["title", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
+	column_headings = ["title", "publish", "resource", "extent", "date", "language", "repository", "creator", "scope", "restrictions"]
 	writer.writerow(column_headings)
 
-	print "Getting a list of resources"
-	resourceIds = requests.get(repositoryBaseURL + "/resources?all_ids=true", headers=headers)
-	print "Writing rows"
-	for resourceId in resourceIds.json():
-			resource = (requests.get(repositoryBaseURL + "/resources/" + str(resourceId), headers=headers)).json()
-			if resource["publish"]:
+	#Checking ALL resources
+	if unpublished_response:
+		if uniqueid_response:
+			unique_id = raw_input("Enter the beginning of the resource ID you wish to include in the script: ")
+			print "Evaluating all resources containing", unique_id,"in their resource ID"
+			for resourceId in resourceIds.json():
+				resource = (requests.get(repositoryBaseURL + "/resources/" + str(resourceId), headers=headers)).json()	
+				if unique_id in resource["id_0"]:
+					makeRow(resource)
+					writer.writerow(row)
+				else:
+					pass
+		else:
+			print "Evaluating all resources"
+			for resourceId in resourceIds.json():
+				resource = (requests.get(repositoryBaseURL + "/resources/" + str(resourceId), headers=headers)).json()	
 				makeRow(resource)
 				writer.writerow(row)
+				
+	#Checking ONLY published resources
+	else:
```

Comment:
@helrond if I change this wouldn't it let ANY response go through since it checking for an _existence_ of a response or a certain value? Defeating the purpose of the else clause i will add below since it will take any response. Unless you want me to change it to `elif unpublished_response == ("n" or "no")` and leave out where it check for the existence of an answer